definitions/npm/source-map_v0.7.x/flow_v0.25.x-/source-map_v0.7.x.js

```diff
@@ -0,0 +1,363 @@
+declare module 'source-map' {
+  declare export type SourceMapUrl = string;
+  declare export type StartOfSourceMap = {
```

Comment:
Here same, exactness and immutability would be nice!

https://github.com/flow-typed/flow-typed/blob/master/CONTRIBUTING.md#prefer-immutability
https://github.com/flow-typed/flow-typed/blob/master/CONTRIBUTING.md#prefer-exactness